### PR TITLE
Counting Sort - English version

### PR DESCRIPTION
**Pull request by vbrazo**

### CODE REVIEW

File: en/Sorting Algorithms/Couting Sort.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Counting Sort
+
+#### Problem Statement
+
+Given an unsorted array of n elements, write a function to sort the array
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:5]:
```suggestion
Given an unsorted array of `n` elements, write a function to sort the array.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Approach
+
+- Find out the maximum element (let it be max) from the given array.
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:9]:
```suggestion
- Find out the maximum element (let's call it `max`) from the given array.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+- Initialize an array of length `max+1` with all elements 0 to store the count of the elements in the array.
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:10]:
```suggestion
- Initialize an array of length `max+1` with all elements set to 0 to store the array's count.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+- Store the count of each element at their respective index in count array.
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:11]:
```suggestion
- Store the count of each element at their respective index in the array's count.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+- Store cumulative sum of the elements of the count array. It helps in placing the elements into the correct index of the sorted array.
+- Find the index of each element of the original array in the count array. This gives the cumulative count.
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:13]:
```suggestion
- Find the index of each element of the original array in the array's count. This gives the cumulative count.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+- Place the element at the index calculated and decrease its count by one.
+
+#### Time Complexity
+
+- `O(n+k)`, best-case performance
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:18]:
```suggestion
- `O(n+k)`: best-case performance
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 7]
```diff
@@ -0,0 +1,63 @@
...
+
+- `O(n+k)`, average-case performance
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:20]:
```suggestion
- `O(n+k)`: average-case performance
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 8]
```diff
@@ -0,0 +1,63 @@
...
+
+- `O(n+k)`, worst-case performance - where k is the range of the non-negative key values.
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:22]:
```suggestion
- `O(n+k)`: worst-case performance - where `k` is the range of the non-negative key values.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 9]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Space Complexity
+
+`O(n+k)`, worst-case performance
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:26]:
```suggestion
`O(n+k)`: worst-case performance
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+#### Founder's Name
+
+- Harold H. Seward
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:30]:
```suggestion
- Harold H. Seward.
```

File: en/Sorting Algorithms/Couting Sort.md [hunk 11]
```diff
@@ -0,0 +1,63 @@
...
+  for j <- size down to 1
+    restore the elements to array
+    decrease count of each element restored by 1
+```
+
+#### Code Implementation Links
+
+- [C](https://github.com/TheAlgorithms/C/blob/master/sorting/counting_sort.c)
+- [C++](https://github.com/TheAlgorithms/C-Plus-Plus/blob/master/sorting/counting_sort.cpp)
+- [Javascript](https://github.com/TheAlgorithms/Javascript/blob/master/Sorts/CountingSort.js)
```
Comment (Panquesito7) [inline en/Sorting Algorithms/Couting Sort.md:52]:
```suggestion
- [JavaScript](https://github.com/TheAlgorithms/Javascript/blob/master/Sorts/CountingSort.js)
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]
👍 Good work.

**Review by Panquesito7 — APPROVED** [review]
Great work, @vbrazo. Thanks for contributing! 👍 🎉
@appgurueu, would you like to review this PR, too? 🙂

**Review by appgurueu — CHANGES_REQUESTED** [review]
Please define `k`.

**Review by Panquesito7 — APPROVED** [review]

**Pull request merged**